webapp/channels/src/packages/mattermost-redux/src/selectors/entities/teams.ts

```diff
@@ -145,10 +145,13 @@ export const getCurrentRelativeTeamUrl: (state: GlobalState) => string = createS
 
 export function getRelativeTeamUrl(state: GlobalState, teamId: string): string {
     const team = getTeam(state, teamId);
+    if (!team) {
+        return '/';
```

Comment:
I'm not sure about how we're handling this one since this seems like it'd likely still result in incorrect behaviour. This selector's only used in one place in `websocket_actions.jsx`, so maybe we could provide better handling for this case there?

Reply:
I started using it also in `actions/views/channel.ts`.

That being said, not sure what better handling could we make. We could return the empty string, and handle that not performing the navigation, but 0/5.